src/components/LandingPage/FAQ/FAQ.tsx

```diff
@@ -24,19 +25,28 @@ export const FAQDropDown: React.FC<FAQDropDownProps> = (props) => {
   }
   return (
     <a
-      className="collapse border border-purple_hover rounded-box"
+      className="border collapse border-purple_hover rounded-box"
```

Comment:
you don't need the ```border-purple_hover``` anymore since the border color is pink now, can just be
```html
 <a className="collapse border-2 border-pink_200 rounded-box">
```

Reply:
bet, I will look into it and make those changes